src/main/scala/org/globalforestwatch/summarystats/firealerts/FireAlertsAnalysis.scala

```diff
@@ -3,29 +3,17 @@ package org.globalforestwatch.summarystats.firealerts
 import java.time.LocalDateTime
 import java.time.format.DateTimeFormatter
 
-import geotrellis.vector.{Feature, Geometry}
-import org.apache.spark.HashPartitioner
+import geotrellis.vector.Feature
 import org.apache.spark.rdd.RDD
-import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.{DataFrame, SparkSession}
 import org.globalforestwatch.features.{FeatureDF, FeatureFactory, FeatureId}
-import org.globalforestwatch.summarystats.firealerts.FireAlertsRDD.SUMMARY
 import org.globalforestwatch.util.Util._
-import org.datasyslab.geospark.spatialRDD.{PointRDD, SpatialRDD}
-import org.datasyslab.geospark.enums.{GridType, IndexType}
-import org.datasyslab.geospark.spatialOperator.JoinQuery
-import java.util.HashSet
-
 import cats.data.NonEmptyList
-
-import collection.JavaConverters._
-import scala.reflect.ClassTag
-import org.apache.spark.api.java.JavaRDD
-import org.datasyslab.geosparksql.utils.Adapter
+import geotrellis.vector
 
 object FireAlertsAnalysis {
-  def apply(featureRDD: RDD[Feature[Geometry, FeatureId]],
+  def apply(featureRDD: RDD[Feature[vector.Geometry, FeatureId]],
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=5902460)